Cargo.toml

```diff
@@ -65,8 +65,11 @@ utxo = { path = "utxo"}
 [profile.release]
 debug-assertions = true
 overflow-checks = true
+lto = true
```

Comment:
Should we also use `codegen-units = 1`?

Reply:
https://doc.rust-lang.org/rustc/codegen-options/index.html#codegen-units

Reply:
Yeah, I'm not sure... builds may become release slow...

Reply:
OK, I didn't see that we're building everything in the release mode on CI.